Docker/Dockerfile

```diff
@@ -1,44 +1,49 @@
-FROM phusion/baseimage:0.11
+FROM debian:stable-slim
+
 #########################################
 ##        ENVIRONMENTAL CONFIG         ##
 #########################################
 
 # Set correct environment variables
+ENV DEBIAN_FRONTEND noninteractive
 ENV HOME="/root"
 
-# Use baseimage-docker's init system
-CMD ["/sbin/my_init"]
-
-# Install airplay
-RUN mkdir /var/run/dbus
-RUN apt-get update
-RUN apt-get install -y wget perl tzdata pv libcrypt-openssl-rsa-perl libio-socket-inet6-perl libwww-perl avahi-utils libio-socket-ssl-perl dbus avahi-daemon
-ADD libnet-sdp-perl_0.07-1_all.deb /tmp
-RUN dpkg -i /tmp/libnet-sdp-perl_0.07-1_all.deb
-
-ADD avahi-daemon.conf /etc/avahi/avahi-daemon.conf
-
-# Fix avahi-daemon not working without dbus
-# RUN sed -i -e "s#\#enable-dbus=yes#enable-dbus=false#g" /etc/avahi/avahi-daemon.conf
-
-# Add avahi-daemon to runit
-RUN mkdir /etc/service/avahi-daemon
-ADD avahi-daemon.sh /etc/service/avahi-daemon/run
-RUN chmod +x /etc/service/avahi-daemon/run
+#########################################
+##         INSTALL PACKAGES            ##
+#########################################
 
-# Fix spamming in logs
-RUN echo "export NO_AT_BRIDGE=1" >> /etc/bash.bashrc
+RUN apt-get update  && \
+apt-get install -y wget perl tzdata pv libwww-perl avahi-utils libio-socket-ssl-perl libnet-ssleay-perl curl supervisor && \
+apt-get clean -qy && \
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Add start script
+COPY supervisor/supervisord.conf /etc/supervisor/supervisord.conf
 
 #########################################
 ##         RUN INSTALL SCRIPT          ##
 #########################################
 
 COPY install.sh /tmp/
-RUN chmod +x /tmp/install.sh; sync; /tmp/install.sh; sync; rm /tmp/install.sh
+RUN chmod +x /tmp/install.sh
+RUN /tmp/install.sh
+
+RUN mkdir -p /config /var/log/supervisor
+
+RUN mkdir /music && \
+  chown -R squeezeboxserver:users /config /music
  
 #########################################
 ##         EXPORTS AND VOLUMES         ##
 #########################################
 VOLUME /config /music
```

Comment:
Some people may have a separate folder for playlists. Playlist folders are also configurable in LMS, seperate to the media folders. It would be nice to also configure a "/playlist" volume.

Reply:
Ok, lets add also playlist volume. Any other volumes that should we consider?

Reply:
I think that's it. If you look at the first run configuration of LMS, it only asks for a music folder and an optional playlist folder.